### ISSUE TITLE
Packaged types include import from rollup which fails type checking since rollup is only installed as a dev dependency

### ISSUE THREAD

**Issue by melink14** [nguyenbatranvan/vite-multiple-assets#42]
Import from rollup:
https://github.com/nguyenbatranvan/vite-multiple-assets/blob/e10e7a717cc729d9da09e02a86f7c83be03d2715/packages/libs/src/index.ts#L6

Leads to this type error:
```
> tsc --noEmit

node_modules/vite-multiple-assets/dist/index.d.mts:3:41 - error TS2307: Cannot find module 'rollup' or its corresponding type declarations.

3 import { NormalizedOutputOptions } from 'rollup';
                                          ~~~~~~~~


Found 1 error in node_modules/vite-multiple-assets/dist/index.d.mts:3
```

I fixed this in the short term by adding rollup as a top level dev dependency which allows it to work. I hadn't noticed it earlier since I coincidentally had rollup as a transitive dependency.

**Comment by nguyenbatranvan** [comment]
I think this might occur with projects using `pnpm`, right?

**Comment by melink14** [comment]
I'm not sure but I'm using normal npm for my project.

**Comment by nguyenbatranvan** [comment]
> I'm not sure but I'm using normal npm for my project.

hmmm!! This issue will be resolved if I move `rollup` to `dependencies` instead of `devDependencies`.

**Comment by nguyenbatranvan** [comment]
fixed `2.2.5`
